sql/catalyst/src/main/antlr4/org/apache/spark/sql/catalyst/parser/SqlBase.g4

```diff
@@ -136,20 +136,22 @@ statement
     | ALTER TABLE multipartIdentifier
         ADD (COLUMN | COLUMNS)
         '(' columns=qualifiedColTypeWithPositionList ')'               #addTableColumns
+    | ALTER TABLE table=multipartIdentifier
+        RENAME COLUMN
+        from=multipartIdentifier TO to=multipartIdentifier             #renameTableColumn
```

Comment:
`to` should be a single `identifer`. We can only rename a column or a field, but not a field chain.

Reply:
oh, then it should be `errorCapturingIdentifier`.